http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/WsmDao.scala

```diff
@@ -33,10 +31,6 @@ import java.time.ZonedDateTime
 import java.util.UUID
 
 trait WsmDao[F[_]] {
-  def createIp(request: CreateIpRequest, authorization: Authorization)(implicit
```

Comment:
I deleted the methods/models from `WsmDao` around IP, network, and relay namespace which are not used anymore.

Side note: I would _really_ like if Leo migrated to using published Java client libraries for WSM (and Sam) in the future. That would reduce boilerplate code and surface for potential contract-related bugs.

Reply:
That is a very good point, I will create a tech debt ticket for us to address that

Reply:
https://broadworkbench.atlassian.net/browse/IA-4175